Packs/MicrosoftTeams/ReleaseNotes/1_1_17.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+##### Microsoft Teams
+- Added support for 'Informational' severity level
+- Made create cards span the entire width of the message in Teams
+- Fixed a bug with Auto Notifications not working properly
```

Comment:
i think its a bit confusing, as the parameter is now different no?

Reply:
Re-written the rn, ill push and if you have comments you can re-write and ill edit again